src/cli/bin.js

```diff
@@ -15,6 +15,8 @@ updateNotifier({
 yargs
   .commandDir('commands')
   .demand(1)
+  // Temporary alias of {add,cat,get}
+  .commandDir('commands/files')
```

Comment:
What you just said is correct :)

You can replace the comment to

```
Note: These creates an alias of `jsipfs files {add, get, cat}` to `jsipfs {add, get, cat}`. This will stay until https://github.com/ipfs/specs/issues/98 is resolved.
```